ClosedXML/Excel/ConditionalFormats/Save/XLCFDatesOccuringConverter.cs

```diff
@@ -29,7 +30,7 @@ public ConditionalFormattingRule Convert(IXLConditionalFormat cf, int priority,
 
             conditionalFormattingRule.TimePeriod = cf.TimePeriod.ToOpenXml();
 
-            var address = cf.Range.RangeAddress.FirstAddress.ToStringRelative(false);
+            var address = cf.Ranges.First().RangeAddress.FirstAddress.ToStringRelative(false);
```

Comment:
Just checking... shouldn't we use `cf.Range` instead of `cf.Ranges.First()` ?
So the question is, if there is more than one range, should this fail or continue?

Reply:
I just realized that here most likely we should use the top-left cell for all ranges, like [there](https://github.com/ClosedXML/ClosedXML/issues/757#issuecomment-380886527). I will check and if so will add ```BaseCellAddress``` property to XLConditionalFormat.

Reply:
> we should use the top-left cell for all ranges
No, it's OK to get the first cell of the first range.

> shouldn't we use ```cf.Range``` instead of ```cf.Ranges.First()```?
I made ```Range``` returning ```.FirstOrDefault``` instead of ```.Single()``` and reverted these changes. Rules with no ranges in collections are removed during consolidation, so we won't get NullRef here.